brython/pip_install/Makefile

```diff
@@ -0,0 +1,6 @@
+.PHONY: all
```

Comment:
Agree, removing these 2 files. They are not relevant in this context. It came as a reflex, as I tend to create a Makefile in all my projects so that I don't have to remember all the commands I need to run.